doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -270,15 +270,15 @@ The logs are not removed after reading, but when attempting to write new logs to
 
 The diagnostic network and end user logs are designed to be pushed when requested by the user.
 This can result in the same information being passed by multiple APIs, which is usually not desirable behavior.
-Because of this, for the network and the end user logs the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REDIRECT` Kconfig option is enabled by default.
+Because of this, for the network and the end user logs the :ref:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REDIRECT<CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REDIRECT>` Kconfig option is enabled by default.
```

Comment:
```suggestion
Because of this, for the network and the end-user logs the :ref:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REDIRECT<CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REDIRECT>` Kconfig option is enabled by default.
```

Reply:
Applied to a couple of other places in the same file as well.